runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -243,9 +244,29 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals("bank") && event.getKey().equals("removeTutorialButton"))
+		{
+			final Widget button = client.getWidget(WidgetInfo.BANK_TUTORIAL_BUTTON);
+			if (button != null)
+			{
+				button.setHidden(config.removeTutorialButton());
+			}
+		}
+	}
+
 	@Subscribe
 	public void onWidgetLoaded(WidgetLoaded event)
 	{
+		if (event.getGroupId() == WidgetID.BANK_GROUP_ID && config.removeTutorialButton())
+		{
+			final Widget button = client.getWidget(WidgetInfo.BANK_TUTORIAL_BUTTON);
+			button.setHidden(true);
```

Comment:
I guess this can just be `client.getWidget(WidgetInfo.BANK_TUTORIAL_BUTTON).setHidden(true);`